### PR TITLE
DTFS2-5783 - TFM- MIN stage confirmed

### PR DESCRIPTION
**Pull request by Zainzzkk**
# Changes made: 

* Added functionality for MIN submissions to change tfm deal stage to confirmed
* Added submit-to-tfm test
* Updated tfm-api test
* Added constant for abandoned deal on TFM-api

### CODE REVIEW

File: e2e-tests/submit-to-trade-finance-manager/cypress/integration/journeys/portal/submit-MIN-changes-tfm-stage-to-confirmed/submit-MIN-changes-tfm-stage-to-confirmed.spec.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    // TFM stage should be updated
+    //---------------------------------------------------------------
+    cy.forceVisit(TFM_URL);
+
+    cy.tfmLogin(UNDERWRITER_MANAGER_1);
+
+    const tfmDealPage = `${TFM_URL}/case/${dealId}/deal`;
+    cy.forceVisit(tfmDealPage);
+
+    tfmPartials.caseSummary.ukefDealStage().contains('Confirmed');
```
Comment (abhi-markan) [inline e2e-tests/submit-to-trade-finance-manager/cypress/integration/journeys/portal/submit-MIN-changes-tfm-stage-to-confirmed/submit-MIN-changes-tfm-stage-to-confirmed.spec.js:75]: Please use constant.

File: e2e-tests/submit-to-trade-finance-manager/cypress/integration/journeys/portal/submit-MIN-changes-tfm-stage-to-confirmed/submit-MIN-changes-tfm-stage-to-confirmed.spec.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    portalPages.contractReadyForReview.readyForCheckersApproval().click();
+
+    //---------------------------------------------------------------
+    // portal checker submits deal to ukef
+    //---------------------------------------------------------------
+    cy.login(BANK1_CHECKER1);
+    portalPages.contract.visit(deal);
+
+    portalPages.contract.status().invoke('text').then((text) => {
+      expect(text.trim()).to.equal('Ready for Checker\'s approval');
```
Comment (abhi-markan) [inline e2e-tests/submit-to-trade-finance-manager/cypress/integration/journeys/portal/submit-MIN-changes-tfm-stage-to-confirmed/submit-MIN-changes-tfm-stage-to-confirmed.spec.js:54]: Please use constant

**Review by abhi-markan — CHANGES_REQUESTED** [review]
Nice work, I think we need an  `Abandoned` test as well.

File: e2e-tests/submit-to-trade-finance-manager/cypress/integration/journeys/portal/submit-MIN-changes-tfm-stage-to-confirmed/submit-MIN-changes-tfm-stage-to-confirmed.spec.js (hunk 2 above)
Comment (Zainzzkk) [reply e2e-tests/submit-to-trade-finance-manager/cypress/integration/journeys/portal/submit-MIN-changes-tfm-stage-to-confirmed/submit-MIN-changes-tfm-stage-to-confirmed.spec.js:54]: done

File: e2e-tests/submit-to-trade-finance-manager/cypress/integration/journeys/portal/submit-MIN-changes-tfm-stage-to-confirmed/submit-MIN-changes-tfm-stage-to-confirmed.spec.js (hunk 1 above)
Comment (Zainzzkk) [reply e2e-tests/submit-to-trade-finance-manager/cypress/integration/journeys/portal/submit-MIN-changes-tfm-stage-to-confirmed/submit-MIN-changes-tfm-stage-to-confirmed.spec.js:75]: done

File: e2e-tests/trade-finance-manager/cypress/fixtures/constants.js [hunk 3]
```diff
@@ -38,9 +38,20 @@ const FACILITY_TYPE = {
   CONTINGENT: 'Contingent',
 };
 
+const DEAL_STAGE_TFM = {
```
Comment (IgnatG) [inline e2e-tests/trade-finance-manager/cypress/fixtures/constants.js:41]: we already had this inside the e2e-fixtures folder

Comment (IgnatG) [reply e2e-tests/trade-finance-manager/cypress/fixtures/constants.js:41]: actually, we didn't :D

**Review by IgnatG — APPROVED** [review]

**Pull request merged**